shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -406,8 +406,9 @@ - (void)viewWillAppear:(BOOL)animated {
 
   // Only recreate surface on subsequent appearances when viewport metrics are known.
   // First time surface creation is done on viewDidLayoutSubviews.
-  if (_viewportMetrics.physical_width)
+  if (_viewportMetrics.physical_width) {
```

Comment:
I prefer these too. Lets sneak these in while Jason is on vacation :)

Reply:
It isn't my opinion, I'm just following Google's C++ style guide.  Last I checked Google is still writing my paycheck ;)